components/data-feed-service/Makefile

```diff
@@ -1,13 +1,10 @@
-GOFILES=$(shell go list ./... | grep -v '/vendor|asset\.go|proto/' | sed 's/^_//')
-GOPATH=$(shell go env GOPATH)
```

Comment:
Based on the pattern "GOPATH should go away" also consider:
components/cereal-service/Makefile
components/compliance-service/Makefile

These two files are unique in their use of GOPATH without actually defining it; is that a problem?
components/nodemanager-service/Makefile
components/secrets-service/Makefile